### ISSUE TITLE
更新1.8beta10后，解锁状态下无法正常启动Steam

### ISSUE THREAD

**Issue by chaimchen** [st2024/Steamtools#10]
![image](https://github.com/st2024/Steamtools/assets/49957941/669ebd3a-238b-4640-8f6d-a15768686499)
加载后Steam闪退

**Comment by nos4r3** [comment]
+1 same

**Comment by Xiuqi-Yang** [comment]
yeah,please check this version,the situation is the same with friends above me

**Comment by chaimchen** [comment]
> yeah,please check this version,the situation is the same with friends above me

like fixed
